pkg/proc/bininfo.go

```diff
@@ -2296,9 +2315,7 @@ func (bi *BinaryInfo) loadDebugInfoMaps(image *Image, debugInfoBytes, debugLineB
 
 			if cu.lineInfo != nil {
 				if imageMod == nil {
-					if mod, err := gopmod.Load(filepath.Dir(image.Path)); err == nil {
-						imageMod = mod
-					}
+					imageMod = getImageMod(image.Path)
```

Comment:
gopmod.Load use https://pkg.go.dev/github.com/goplus/mod#FindGoMod to lookup `go.mod`. has it bug?